Agents/GFAAgent/src/main/java/uk/ac/cam/cares/jps/agent/gfaagent/GFACalculation.java

```diff
@@ -0,0 +1,54 @@
+package uk.ac.cam.cares.jps.agent.gfaagent;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.apache.jena.vocabulary.AS;
+import org.json.JSONArray;
+
+import uk.ac.cam.cares.jps.base.query.RemoteRDBStoreClient;
+import uk.ac.cam.cares.jps.base.exception.JPSRuntimeException;
+public class GFACalculation {
+
+    private final String dbUrl;
+    private final String user;
+    private final String password;
+
+    private RemoteRDBStoreClient postgisClient;
+
+    public GFACalculation (String postgisDb, String postgisUser, String postgisPassword){
+        this.dbUrl = postgisDb;
+        this.user = postgisUser;
+        this.password = postgisPassword;
+
+        this.postgisClient = new RemoteRDBStoreClient(dbUrl, user, password);
+    }
+    
+    /******************************************** */
+    /* Calculate GFA and store in citydb.cityobject_genericattrib */
+    /**********************************************/ 
+
+    public void calculationGFA(){
+        try (Connection srcConn = postgisClient.getConnection()) {
+            try (Statement stmt = srcConn.createStatement()) {
+                stmt.executeUpdate(gfaSQLInsert);
+            }
+        }catch (SQLException e) {
+            throw new JPSRuntimeException("Error connecting to source database: " + e);
+        }  
+    }
+
+    private static final String gfaSQLInsert = "INSERT INTO cityobject_genericattrib (attrname, realval, cityobject_id)\n" +
```

Comment:
Please move this to line 19 as all fields should be on top. Either that or create a new static class to store and access this query.